tests/nlu/base/test_server.py

```diff
@@ -21,10 +22,10 @@ def app(tmpdir_factory):
     :param component_builder:
     :return:
     """
-
```

Comment:
I'm actually not sure which pattern we follow here (empty line or not after docstring). Makes me even more excited for black 😄

Reply:
:) will remove the empty line for now